.github/workflows/tests.yml

```diff
@@ -32,10 +32,21 @@ jobs:
         # We need to set the safe git directory as formatting relies on git-ls
         # See actions/checkout#766
       - name: "Set the GH workspace as a safe git directory"
-        run: git config --global --add safe.directory "$GITHUB_WORKSPACE"
+        run: |
+          git config --global --add safe.directory "$GITHUB_WORKSPACE"
+          git config --global --add safe.directory "$GITHUB_WORKSPACE/faabric"
+          git config --global --add safe.directory "$GITHUB_WORKSPACE/cpp"
+          git config --global --add safe.directory "$GITHUB_WORKSPACE/python"
       # Formatting checks
       - name: "Code formatting check"
-        run: ./bin/inv_wrapper.sh format-code
+        run: ./bin/inv_wrapper.sh format-code --check
```

Comment:
We were missing the `--check` flag, so formatting errors were slipping through....